cmd/prometheus/main.go

```diff
@@ -322,9 +325,15 @@ func main() {
 	a.Flag("web.enable-admin-api", "Enable API endpoints for admin control actions.").
 		Default("false").BoolVar(&cfg.web.EnableAdminAPI)
 
+	// TODO(bwplotka): Moveo all remote receive flags to config.
```

Comment:
Typo to fix

Reply:
should we resolve this todo prior to merging? we need to resolve it before the next release at least

Reply:
I don't think so, it's an orthogonal feature (runtime changing receive vs remote-write). 

We don't need this even before next release UNLESS we want to avoid deprecating those flags and keeping them until 3.0. Added issue to discuss, I just don't see users asking for this, so would skip for now personally. But let's consider? https://github.com/prometheus/prometheus/issues/14410